.github/workflows/ci_docker.yml

```diff
@@ -90,7 +97,10 @@ jobs:
           fi
 
       - name: Push Docker images to Docker Hub
-        if: ${{ success() && steps.match-branch.outputs.match == 'true' }}
+        if: |
+          success()
+          && steps.check-workflow-diff.outputs.changed_only == 'no'
+          && steps.match-branch.outputs.match == 'true'
```

Comment:
If only the workflow file (`.github/workflows/ci_docker.yml`) is changed, there is no need to publish images.